feature_engine/discretisation/arbitrary.py

```diff
@@ -107,36 +120,32 @@ def fit(self, X: pd.DataFrame, y: Optional[pd.Series] = None):
         return self
 
     def transform(self, X: pd.DataFrame) -> pd.DataFrame:
-        """Sort the variable values into the intervals.
```

Comment:
question: the docstrings were removed. Do they still appear in the docs? otherwise we need to re-introduce